README.md

```diff
@@ -1,6 +1,14 @@
 # unity8-notch-hax
 Lomiri (formely Unity 8) patches for improved display notch and rounded corners support on devices such as the Volla Phone.
 
+## Supported devices
+
+|Device              |Status                |
+|--------------------|----------------------|
+| Volla Phone        | Supported            |
+| OnePlus 6          | Supported            |
+| OnePlus 6T         | Untested             |
```

Comment:
Not sure how I feel about adding support for an untested device, some 6T user could run the script with `./apply-notch-hax.sh OnePlus6` and report back on how it looks; assuming no modification is needed from the regular 6, `OnePlus6T.diff` could even become a symlink to `OnePlus6.diff` (I've not tested if this works, but it can be done later).

The table could also be simplified down to a list:
```md
## Supported devices
* Volla Phone
* OnePlus 6
```

Reply:
Sure, I think I'll drop it entirely and let you handle it :P

Reply:
A symlink wouldn't work with how the script is currently (allowing just the script to be pulled without the whole repo)